structure/validator.go

```diff
@@ -223,3 +224,7 @@ type Array interface {
 
 	Using(usingFunc ArrayUsingFunc) Array
 }
+
+type Bytes interface {
+	NotEmpty() Bytes
```

Comment:
Please add Exists, NotExists, and Empty for consistency with other validators. This is the "minimum bar" for a new validator.

Reply:
There's nothing at all that would use those functions, and there's no documentation indicating any "minimum bar". I'd prefer not to add code that isn't going to be used, and has no documented requirement, tests, name or ... anything?

Reply:
Okay.